app/src/main/java/com/example/mywatchlist/Movies.kt

```diff
@@ -5,55 +5,27 @@ import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.recyclerview.widget.GridLayoutManager
+import com.example.mywatchlist.adapters.ItemAdapterForMovies
+import com.example.mywatchlist.databinding.FragmentMoviesBinding
+import dagger.hilt.android.AndroidEntryPoint
 
-// TODO: Rename parameter arguments, choose names that match
-// the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
-private const val ARG_PARAM1 = "param1"
-private const val ARG_PARAM2 = "param2"
-
-/**
- * A simple [Fragment] subclass.
- * Use the [Movies.newInstance] factory method to
- * create an instance of this fragment.
- */
+@AndroidEntryPoint
 class Movies : Fragment() {
```

Comment:
The name should have Fragment appended to it per contention, as in `MoviesFragment`

Reply:
Done